ws-client/src/client.rs

```diff
@@ -155,9 +175,9 @@ impl<'a> WsClientBuilder<'a> {
 		self
 	}
 
-	/// Set origin header to pass during the handshake.
-	pub fn origin_header(mut self, origin: Cow<'a, str>) -> Self {
-		self.origin_header = Some(origin);
+	/// Set custom headers to pass during the handshake.
```

Comment:
In Soketto there aren't any checks to see whether the headers that are set conflict with the oens it sets internally; I wonder whether it's worth noting that it's the users responsibility here to make sure that headers don't conflict

Reply:
Oh I missed this comment. Can we document the headers set internally here so users know what to not conflict with?